### ISSUE TITLE
Changing angle

### ISSUE THREAD

**Issue by fgnievinski** [VoltaicEngineering/Solar-Panel-Brackets#1]
If I need the bracket to have an angle different than 45 degrees at my location, would it be okay just to bend this bracket?

**Comment by VoltaicEngineering** [comment]
yes, if you have a press brake or another tool to bend the bracket, you can bend it manually to adjust the angle.
